dashboard/app/models/lesson.rb

```diff
@@ -23,17 +23,19 @@
 
 # Ordered partitioning of script levels within a script
 # (Intended to replace most of the functionality in Game, due to the need for multiple app types within a single Game/Stage)
-class Stage < ActiveRecord::Base
+class Lesson < ActiveRecord::Base
   include LevelsHelper
   include SharedConstants
   include Rails.application.routes.url_helpers
   include SerializedProperties
 
-  has_many :script_levels, -> {order('position ASC')}, inverse_of: :stage
-  has_one :plc_learning_module, class_name: 'Plc::LearningModule', inverse_of: :stage, dependent: :destroy
-  belongs_to :script, inverse_of: :stages
+  has_many :script_levels, -> {order('position ASC')}, inverse_of: :lesson, foreign_key: 'stage_id'
+  has_one :plc_learning_module, class_name: 'Plc::LearningModule', inverse_of: :lesson, foreign_key: 'stage_id', dependent: :destroy
```

Comment:
I love this approach to renaming the associations without having to rename the DB table columns. Nice choice! We can wait and see if it will become worth renaming the tables... but I am guessing that it is unlikely.